src/main/java/org/apache/sling/servlets/post/AbstractPostOperation.java

```diff
@@ -127,6 +135,7 @@ public void run(final SlingHttpServletRequest request,
             }
 
             // fail if any of the base paths (before the postfix) which had a postfix are contained in the modification set
+            log.debug("Fail if any of the base paths (before the postfix) which had a postfix are contained in the modification set");
```

Comment:
What is the purpose of this statement? Is it just for the purpose of having a timestamp (assuming the log is properly configured to write mili/micro-second level timestamps)?